backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -386,4 +386,7 @@ SearchException conflictingFieldTypesForProjection(String absoluteFieldPath,
 			value = "Multiple conflicting types for identifier: '%1$s' vs. '%2$s'.")
 	SearchException conflictingIdentifierTypesForPredicate(ToDocumentIdentifierValueConverter<?> component1,
 			ToDocumentIdentifierValueConverter<?> component2, @Param EventContext context);
+
+	@Message(id = ID_OFFSET_3 + 50, value = "Error on a previous work exception")
+	SearchException previousWorkException(@Cause Throwable throwable);
```

Comment:
I suppose you added this to satisfy some checkstyle rule or similar?

I'm not against it, but this method should return `PreviousWorkException`. We actually use `instanceof` to discriminate between these exceptions and other exceptions, so this is important. See where this type is referenced.

I'm a bit surprised this didn't fail the tests, by the way... maybe a test is missing?

Reply:
You're right, indeed it does not make sense using a log for an exception that is supposed to be catch and not throw to the outside. This code is not covered by test yet. This is the commit to handle the single work.